sentry_sdk/integrations/trytond.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import sentry_sdk
+import sentry_sdk.integrations
+import sentry_sdk.integrations.logging
+
+from trytond.exceptions import TrytonException as TrytondBaseException  # type: ignore
+from trytond.wsgi import app  # type: ignore
+
+
+from sentry_sdk._types import MYPY
+
+if MYPY:
+    from typing import Tuple
+
+# TODO: trytond_worker intergation
+
+
+class TrytondWSGIIntegration(sentry_sdk.integrations.Integration):
+    identifier = "trytond_wsgi"
+
+    def __init__(self):  # type: () -> None
+        pass
+
+    @staticmethod
+    def setup_once():  # type: () -> None
+        def error_handler(e):  # type: (Exception) -> None
+            if isinstance(e, TrytondBaseException):
+                return
+            else:
+                sentry_sdk.capture_exception(e)
+
+        # Expected error handlers signature was changed
+        # when the error_handler decorator was introduced
+        # in Tryton-5.4
+        if hasattr(app, "error_handler"):
+
+            @app.error_handler
+            def _(app, request, e):  # type: ignore
+                error_handler(e)
+
+        else:
+            app.error_handlers.append(error_handler)
+
+
+class TrytondSentryHandler(logging.NullHandler):
```

Comment:
@untitaker I am very interested in the insights of sentry-python members about this kind of handlers.

If I think about it, this has nothing to do with any particular framework, yet I was not capable of finding anything similar in the documentation nor the code base.

No rush, I just need some light here.

Reply:
Where is this handler used? It seems to me its purpose is to be called with certain logger names that we cannot know statically, but I am not sure.

Ideally we would never call `init()` from an integration, as the integration would have to provide a way to pass additional configuration to `init()` if the user needs to do that (forwarding of `**kwargs` is not really great)

Reply:
IIUC this is not required if we use [Load wsgi middleware from configuration](https://bugs.tryton.org/issue8807)

Reply:
Please just monkeypatch the wsgi application instead of injecting config values

Reply:
This handler is *NOT* for the wsgi application. This handler is *NOT* a SentrySDK integration. This handler is to fool custom commandline tools shipped with the Tryton framework, namely trytond-admin, trytond-cron and trytond-worker. I only tested it with trytond-cron.

In order not to write wrappers for every tool, we intend to inject a `sentry_sdk.init` call inside a null handler that should be instantiated once and plugged anywhere (it i s a NullHandler!). The following logconf file can be used:

```
[loggers]
keys = root

[handlers]
keys = console, sentry

[formatters]
keys =

[logger_root]
level = INFO
handlers = console, sentry

[handler_console]
class = StreamHandler
args = (sys.stdout,)

[handler_sentry]
class=sentry_sdk.integrations.trytond.TrytondSentryHandler
args = (
    'https://37a9a6ea994a4dfe81ef0f278ab779a1@sentry.io/1824945',
    ('list', 'of', 'silenced', 'logger', 'qualnames'))
```
Then run `trytond-cron --logconf logconf.cfg --database <dbname>`

I am sorry for the misunderstanding. Obviously this PR will include adequate documentation before it is integrated!

Reply:
@untitaker my point is that this could be a not-trytond-specific feature, shipped with the core of SentrySDK so that any framework or application could benefit from it to inject `sentry_sdk.init` calls in any tools that don't allow the execution of arbitrary start-up code but use the python logging configuration system.
My fear is that it resembles the behaviour of the deprecated raven package so I am very interested in your insights.

Reply:
I understand the desire for this shortcut but I don't believe it will work sufficiently. See usage of `before_send` and more advanced passing of arbitrary python objects to `init()`

Reply:
The handler relies on the standard Sentry LoggingIntegration being initialized by default, which will already spy on logger calls. But any later call to this handler will do nothing since it inherits a standard NullHandler!